org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/handlers/JDTLanguageServer.java

```diff
@@ -463,6 +467,16 @@ public void projectConfigurationUpdate(TextDocumentIdentifier param) {
 		return handler.resolveClasspaths(param);
 	}
 
+	/* (non-Javadoc)
+	 * @see org.eclipse.jdt.ls.core.internal.JavaProtocolExtensions#buildWorkspace(java.lang.String)
+	 */
+	@Override
+	public CompletableFuture<Object> buildWorkspace(String type) throws CoreException {
```

Comment:
If here it throws CoreException, what happens? LS eat it and exited?
i think buildWorkspace command should return build status.
If there are exception, then return failed status, else return a succeed status.